pkg/services/pluginsintegration/plugincontext/plugincontext.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/grafana/grafana/pkg/services/pluginsintegration/pluginstore"
 )
 
+// TODO: Delete this?
```

Comment:
not for this PR either but should we remove this error and use `plugins.ErrPluginNotRegistered` instead?

Reply:
Done in https://github.com/grafana/grafana/pull/74997